mailbag/derivatives/eml.py

```diff
@@ -0,0 +1,62 @@
+# This is an example derivative, meant to show how
+# to hook up a real parser
+
+# Does nothing currently
+from os.path import join
+import mailbag.helper as helper
+import os,glob
+import mailbox
+from mailbag.email_account import EmailAccount
+from structlog import get_logger
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from email import generator
+from mailbag.derivative import Derivative
+#from mailbag.controller import Controller
+
+log = get_logger()
+class ExampleDerivative(Derivative):
+    derivative_name = 'eml'
+
+    def __init__(self,email_account, **kwargs):
+        print("Setup account")
+        super()
+
+    def do_task_per_account(self):
+        print(self.account.account_data())
+
+    def do_task_per_message(self, message, args):
+        a = message.Message
+        print("Message",args.directory)
+        print(args.mailbag_name)
+
+
+
+        msg = MIMEMultipart('alternative')
+
+        msg['Subject'] = message.Subject
+        msg['From'] = message.From
+        msg['To'] = message.To
+        msg['Cc'] = message.Cc
+        msg['Bcc'] = message.Bcc
+
+
+        if(message.Message is not None):
```

Comment:
You shouldn't have to attach `message.Message`, as its a full email message object. If `message.Message` is not None, you should be able to just
```
generator = email.generator.Generator(out_file)
generator.flatten(message.Message)
```